packages/emoji-mart-website/index.html

```diff
@@ -75,7 +75,7 @@
             <option value="hi">हिन्दी</option>
             <option value="it">Italiano</option>
             <option value="ja">日本語</option>
```

Comment:
@cyd5538 In that sense, I think “ja” should be changed to “jp”?

Reply:
So I think it makes sense to consider "ko" or all the rest of the languages ​​also change.

Reply:
@oinochoe  That's a good comment!